ballet/update.py

```diff
@@ -0,0 +1,84 @@
+import funcy
+import git
+import json
+import os
+import tempfile
+import yaml
+
+from cookiecutter.main import cookiecutter
+from cookiecutter.prompt import prompt_for_config
+
+from ballet.compat import pathlib, safepath
+from ballet.util.log import logger
+from ballet.quickstart import generate_project
+
+PROJECT_CONTEXT_PATH = (
+    pathlib.Path(__file__).resolve().parent.joinpath('project_template', 'cookiecutter.json')
+)
+
+REPLAY_PATH = (
```

Comment:
Should try to use the `.cookiecutter_replay.json` in the project directory, and try to never touch `~`. You're already creating `.cookiecutter_replay.json` in the post_gen_hook anyway

Reply:
This is not used to collect the context from our ballet repo, but rather to get the previous replay, and save it before our new cookiecutter overwrites it. _create_replay has the workflow:
- if an old replay exists at replay_path, save it as json
- cookiecutter generation (overwrites replay)
- if old replay was saved, write it back to replay.